--- configure	2013-09-03 16:39:12.691938210 +0400
+++ /root/configure	2013-09-03 16:38:56.712020322 +0400
@@ -3628,10 +3628,10 @@
 
 
 PHP_CONFIG=$PHP_PHP_CONFIG
+EXTENSION_DIR="${prefix}`$PHP_CONFIG --extension-dir 2>/dev/null`"
 prefix=`$PHP_CONFIG --prefix 2>/dev/null`
 phpincludedir=`$PHP_CONFIG --include-dir 2>/dev/null`
 INCLUDES=`$PHP_CONFIG --includes 2>/dev/null`
-EXTENSION_DIR=`$PHP_CONFIG --extension-dir 2>/dev/null`
 PHP_EXECUTABLE=`$PHP_CONFIG --php-binary 2>/dev/null`
 
 if test -z "$prefix"; then
