--- /tmp/bind-9.9.2-P1/bin/named/Makefile.in	2013-02-16 18:07:52.734176205 +0400
+++ bin/named/Makefile.in	2013-02-16 18:09:46.163586980 +0400
@@ -28,10 +28,10 @@
 #
 # Add database drivers here.
 #
-DBDRIVER_OBJS =
-DBDRIVER_SRCS =
-DBDRIVER_INCLUDES =
-DBDRIVER_LIBS =
+DBDRIVER_OBJS = ldapdb.@O@
+DBDRIVER_SRCS = ldapdb.c
+DBDRIVER_INCLUDES = -I/usr/local/include
+DBDRIVER_LIBS = -L/usr/local/lib -lldap -llber -lresolv
 
 DLZ_DRIVER_DIR =	${top_srcdir}/contrib/dlz/drivers
 
